modules/ocf_irc/files/anope/chanserv.conf

```diff
@@ -0,0 +1,484 @@
+# Comments trimmed for easier editing.
```

Comment:
Is there any way we can avoid committing this entire massive file? I feel like we only made a couple minor edits. Committing entire files is pretty fragile and has been cause for much pain in the past.

It looks like anope supports includes; maybe you can append an include line to the file, and put our small number of changes in the included file?

Alternatively, if it's just a couple lines changed, `sed` isn't a terrible option. Normally augeas would be preferred, but this looks like a complicated custom config format, so probably not an option here.